dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -157,6 +157,14 @@
       <attribute name="jvm.args" default="${tests.jvm.argline}"/>
       <element name="additional-args" optional="true"/>
     <sequential>
+      <!-- make sure no elasticseatch instance is currently running and listening on the port we need -->
+      <fail message="This test expects port @{es.http.port} to be free but an elasticsearch instance already running and listening on port @{es.http.port}.
+      Maybe the last test run did not manage to shut down the node correctly?
+      You must kill it before tests can run.">
+        <condition>
+          <http url="http://localhost:@{es.http.port}"></http>
```

Comment:
to be a little more robust, maybe use ```<socket ...>``` instead of `http`? With http, it might be able to connect, but if it gets e.g. a 400 or 500 error, the condition will be set to false, since http was "unsuccessful".